src/main/java/task1withAI/Symbol.java

```diff
@@ -0,0 +1,41 @@
+package task1withAI;
+
+public enum Symbol {
+    X {
+        @Override
+        public String toString() {
+            return "X";
+        }
+
+        @Override
+        public Symbol oppose() {
+            return O;
+        }
+    },
+
+    O {
+        @Override
+        public String toString() {
+            return "O";
+        }
+
+        @Override
+        public Symbol oppose() {
+            return X;
+        }
+    },
+
+    SPACE {
+        @Override
+        public String toString() {
+            return "_";
+        }
+
+        @Override
+        public Symbol oppose() {
+            return null;
+        }
+    };
+
+    public abstract Symbol oppose();
```

Comment:
Why do you need this method?

Reply:
Because I reworked this project from simple tic-tac-toe and this method served to change the symbol to the opposite. Thanks. I deleted this method.